test/WebJobs.Script.Tests.Integration/Host/StandbyManagerTests.cs

```diff
@@ -32,74 +31,66 @@ namespace Microsoft.Azure.WebJobs.Script.Tests
     [Trait(TestTraits.Group, TestTraits.StandbyModeTests)]
     public class StandbyManagerTests : IDisposable
     {
-        private readonly ScriptSettingsManager _settingsManager;
+        private const string _expectedHostId = "standby-test-site-name";
         private TestLoggerProvider _loggerProvider;
         private HttpClient _httpClient;
         private TestServer _httpServer;
-        private string _expectedHostId = "update when tests fixed";
-        private ScriptApplicationHostOptions _webHostOptions;
-        private string _testRootPath;
+        private readonly string _testRootPath;
 
         public StandbyManagerTests()
         {
-            _settingsManager = ScriptSettingsManager.Instance;
             _testRootPath = Path.Combine(Path.GetTempPath(), "StandbyManagerTests");
-            ResetEnvironment();
             CleanupTestDirectory();
         }
 
-        [Fact(Skip = "facaval - investigating")]
+        [Fact]
         public async Task StandbyMode_EndToEnd()
         {
             var vars = new Dictionary<string, string>
             {
                 { EnvironmentSettingNames.AzureWebsitePlaceholderMode, "1" },
-                { EnvironmentSettingNames.AzureWebsiteContainerReady, null },
                 { EnvironmentSettingNames.AzureWebsiteSku, "Dynamic" },
-                { EnvironmentSettingNames.AzureWebsiteHomePath, null },
                 { EnvironmentSettingNames.AzureWebsiteInstanceId, "87654639876900123453445678890144" },
-                { "AzureWebEncryptionKey", "0F75CA46E7EBDD39E4CA6B074D1F9A5972B849A55F91A248" }
```

Comment:
@mathewc -- I wasn't sure what this AzureWebEncryptionKey was for. I didn't see it used anywhere...

Reply:
The cleanup is less of an issue now that we're no longer setting environment variables, which is nice. But I agree that mimicking the standby environment as much as we can is valuable here.

Reply:
What cleanup are you guys talking about? Test cleanup isn't an issue anymore (since there isn't any) -- are you referring so cleanup of the standby environment?

I want to add comments for all this -- it's confusing seeing a bunch of unused variables.